src/components/Mnemonic.tsx

```diff
@@ -159,7 +163,8 @@ function Mnemonic({
                             className={`input input-mnemonics ${classNames({
                                 'input-error':
                                     errors.phrases?.[index] ||
-                                    (values[index]?.value &&
+                                    (values &&
+                                        values[index]?.value &&
```

Comment:
`values && values[index]?.value` => `values?.[index]?.value`
is better

Reply:
cool thank you